crawl-ref/source/dat/descript/de/ability.txt

```diff
@@ -653,26 +653,25 @@ Opfert die Reinheit von Körper oder Geist und verursacht eine einzige Mutation.
 %%%%
 Sacrifice Words ability
 
-Opfert die Fähigkeit, magische Schriftrollen zu benutzen, während man bedroht
-wird.
+Opfert die Fähigkeit, magische Schriftrollen zu benutzen, während Ihr bedroht
+werdet.
 %%%%
 Sacrifice Drink ability
 
-Opfert die Fähigkeit, Tränke zu trinken, während man bedroht wird.
+Opfert die Fähigkeit, Tränke zu trinken, während Ihr bedroht werdet.
 %%%%
 Sacrifice Essence ability
 
-Opfert die magische Essenz und verursacht eine einzige Mutation.
+Opfert magische Essenz und verursacht eine einzige Mutation.
 %%%%
 Sacrifice Health ability
 
-Opfert die körperlichen Gesundheit, wodurch eine einzige Mutation verursacht
-wird.
+Opfert körperlichen Gesundheit und verursacht eine einzige Mutation.
```

Comment:
That's an oversight. With or without 'die', it should be 'körperliche'. The n is wrong in both cases. The 'die' isn't exactly wrong, it just sounds better without it.